components/brave_wallet_ui/components/desktop/views/accounts/account.tsx

```diff
@@ -402,12 +402,13 @@ export const Account = () => {
         >
           {nonFungibleTokens?.length !== 0 ? (
             <NftGrid>
-              {nonFungibleTokens?.map((nft) =>
+              {nonFungibleTokens?.map((nft: BraveWallet.BlockchainToken) =>
                 <NFTGridViewItem
-                  isHidden={false}
                   key={getAssetIdKey(nft)}
                   token={nft}
                   onSelectAsset={() => onSelectAsset(nft)}
+                  isTokenHidden={false}
+                  isTokenSpam={false}
```

Comment:
are these values supposed to be hard-coded?

Reply:
@josheleonard  Yes, we don't show spam and hidden NFTs in accounts